packages/react-charts/src/components/ChartUtils/chart-resize.tsx

```diff
@@ -57,17 +56,17 @@ import { canUseDOM } from './chart-helpers';
  *   );
  * }
  *
- * @param {React.RefObject} containerRef The container reference to observe
+ * @param {Element} containerRefElement The container reference to observe
  * @param {Function} handleResize The function to call for resize events
  * @return {Function} The function used to unobserve resize events
  */
-export const getResizeObserver = (containerRef: React.RefObject<any>, handleResize: () => void) => {
+export const getResizeObserver = (containerRefElement: Element, handleResize: () => void) => {
```

Comment:
I don't believe `Element` is the correct type (too restrictive) as this can be any `RefObject`. I see that the code editor is using `React.createRef<HTMLDivElement>();`, which is what the chart examples use. 

What was the type issue?